pyproject.toml

```diff
@@ -15,8 +15,8 @@ keywords = ["arq", "prometheus", "metrics"]
 
 [tool.poetry.dependencies]
 python = "^3.8"
-prometheus-client = "^0.14.1"
-arq = "^0.23"
+prometheus-client = ">=0.14.1<1.0.0"
+arq = ">0.23<1.0"
```

Comment:
I think according to poetry they should have a comma:
https://python-poetry.org/docs/dependency-specification/#multiple-requirements

Reply:
Interesting, this works with `poetry==1.11.1`. Will update.